src/Compilers/CSharp/Portable/Lowering/LambdaRewriter/SynthesizedClosureMethod.cs

```diff
@@ -163,6 +165,10 @@ protected override ImmutableArray<TypeSymbol> ExtraSynthesizedRefParameters
         internal override bool IsExpressionBodied => false;
         internal MethodSymbol TopLevelMethod => _topLevelMethod;
 
+        public override ImmutableArray<CSharpAttributeData> GetAttributes() => _originalMethod.GetAttributes();
```

Comment:
>_originalMethod.GetAttributes() [](start = 79, length = 31)

Similar to changes in ```SynthesizedMethodBaseSymbol```, I think it would be better if we somehow checked that we actually want to inherit attributes from the ```BaseMthod```. Also, consider moving the logic to ```SynthesizedMethodBaseSymbol``` so that everything is in one place.